core/domain/suggestion_services_test.py

```diff
@@ -865,12 +865,12 @@ def test_update_translation_suggestion_to_change_translation_html(
         self
     ) -> None:
         exploration = (
-            self.save_new_linear_exp_with_state_names_and_interactions(  # type: ignore[no-untyped-call]
+            self.save_new_linear_exp_with_state_names_and_interactions(
                 'exploration1', self.author_id, ['state 1'], ['TextInput'],
                 category='Algebra'))
         old_content = state_domain.SubtitledHtml(
             'content', '<p>old content html</p>').to_dict()
-        exploration.states['state 1'].update_content(
+        exploration.states['state 1'].update_content(  # type: ignore[no-untyped-call]
```

Comment:
Why does this need to be added here?

Reply:
I hope I answered your question.

Reply:
Gotcha. Makes sense. Thanks for the detailed explanation!